source/api_documentation/transactions_api/affiliate_user.rst

```diff
@@ -1,7 +1,69 @@
+#####################
 Publisher / Affiliate
-=====================
+#####################
 
-.. affiliate_tx_api_page::
-  :page: affiliate_tx_api_page
-  :org_type: affiliate
+URL
+---
 
+The API follows REST conventions. Perform an HTTPS GET to the URL with the format in which you’d like to receive data. The following response formats are supported, where 33 is the affiliate id.
+
+.. list-table::
+  :widths: 8 40
+  :header-rows: 1
+  :class: parameters
+
+  * - Format
+    - Description and URL
+
+  * - csv
+    - Comma-Separated Values, or really Anything-Separated Values (see column_separator= below). Returns an optional header row followed by one row for each transaction, with delimited values for each row. ``https://mynetwork.invoca.net/api/@@TRANSACTION_API_VERSION/affiliates/transactions/33.csv``
+
+  * - xml
+    - Returns an XML document with an array of Transaction elements. ``https://mynetwork.invoca.net/api/@@TRANSACTION_API_VERSION/affiliates/transactions/33.xml``
+
+  * - json
+    - Returns a JSON array of transaction objects. ``https://mynetwork.invoca.net/api/@@TRANSACTION_API_VERSION/affiliates/transactions/33.json``
+
+
+.. include:: intro.rst
+
+``https://yourcompany.invoca.net/api/@@TRANSACTION_API_VERSION/affiliates/transactions/706.csv?limit=500&oauth_token=<YOUR_OAUTH_TOKEN>&from=2015-03-26&to=2015-03-27``
+
+returns 500 rows, grab the last transaction_id (in this example 500) and send request 2:
+
+``https://yourcompany.invoca.net/api/@@TRANSACTION_API_VERSION/affiliates/transactions/706.csv?limit=500&oauth_token=<YOUR_OAUTH_TOKEN>U&from=2015-03-26&to=2015-03-27&start_after_transaction_id=500``
+
+then repeat as necessary to get all call records within date range.
+
+Response
+--------
+
+The data returned has the following fields:
+
+.. include:: affiliate_param_table.rst
+
+.. include:: conversion_reporting_param_table.rst
```

Comment:
I confirmed in the last round that affiliates do not have conversion reporting.. so this line can be removed.  

Please also update the script in tools repo to not override `cpa_enabled?` on affiliate report class (it's impossible to be true based on the actual repo code) https://github.com/Invoca/web/blob/master/app/models/reports/detail/detail_report.rb#L857-L857